src/App.tsx

```diff
@@ -33,13 +44,19 @@ export default function App() {
           pauseOnHover
           theme="dark"
         />
-        <Sidebar />
         <Routes>
           <Route path="/" element={<Home />} />
+          <Route path="/login" element={<Login />} />
+          <Route path="treanding" element={<Treanding />} />
```

Comment:
path -> /routename
Spelling mistake -> Trending and not Treanding

Reply:
@Parul1999 yeah I missed that it is fixed in a later PR "https://github.com/SJTGSHIVAM/evid-mock/pull/4" Thanks for pointing